src/main/kotlin/gg/skytils/skytilsmod/features/impl/misc/ItemFeatures.kt

```diff
@@ -590,6 +590,17 @@ class ItemFeatures {
                 }
             }
         }
+        if (Skytils.config.showCollectionStackSize) {
+            val collectionDisplayName = getDisplayName(item)
+            val collectionMatcher = collectionPattern.matcher(collectionDisplayName)
+            if (collectionMatcher.matches()) {
+                lore?.forEach { line -> //prevent false positives
+                    if (line.contains("Collection")) {
+                        stackTip = collectionMatcher.group(2).toString().romanToDecimal().toString()
+                    }
+                }
+            }
```

Comment:
Purely guessing that there are 3 groups
```suggestion
            collectionPattern.matchEntire(getDisplayName(item))?.destructured?.let { (_, _, level) ->
                lore?.forEach { line ->
                    if (line.contains("Collection")) {
                        stackTip = level.romanToDecimal().toString()
                        return@let
                    }
                }
            }
```

Reply:
nope, two.